integration-cli/docker_cli_build_test.go

```diff
@@ -1202,6 +1202,31 @@ func (s *DockerSuite) TestBuildCopyWildcardNoFind(c *check.C) {
 
 }
 
+func (s *DockerSuite) TestBuildCopyWildcardInName(c *check.C) {
+	name := "testcopywildcardinname"
+	defer deleteImages(name)
+	ctx, err := fakeContext(`FROM busybox
+	COPY *.txt /tmp/
+	RUN [ "$(cat /tmp/\*.txt)" = 'hi there' ]
+	`, map[string]string{"*.txt": "hi there"})
+
+	if err != nil {
+		// Normally we would do c.Fatal(err) here but given that
+		// the odds of this failing are so rare, it must be because
+		// the OS we're running the client on doesn't support * in
+		// filenames (like windows).  So, instead of failing the test
+		// just let it pass. Then we don't need to explicitly
+		// say which OSs this works on or not.
+		return
```

Comment:
do we have logs in the tests?  I can do a fmt.Print if you want and if windows folks are ok with the clutter in their test output.  Just let me know.